### PR TITLE
pegtl: version bump

### PR DESCRIPTION
**Pull request by junghans**
/cc @d-frey

### CODE REVIEW

**Comment by junghans** [comment]
taocpp/PEGTL#64 got fixed, so give me a sec.

File: var/spack/repos/builtin/packages/pegtl/package.py [hunk 1]
```diff
@@ -36,9 +36,15 @@ class Pegtl(Package):
     homepage = "https://github.com/taocpp/PEGTL"
     url      = "https://github.com/taocpp/PEGTL/tarball/1.3.1"
 
-    version('1.3.1', '11efc4beac8f4f5153466d56074e9f0c')
+    version('2.0.0', 'c772828e7188459338a920c21f9896db')
+    version('2.1.4', 'e5288b6968e6e910287fce93dc5557bf')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pegtl/package.py]: Make sure that versions are sorted from greatest to least.

**Comment by junghans** [comment]
Needed for #3852.

File: var/spack/repos/builtin/packages/pegtl/package.py (hunk 1 above)
Comment (junghans) [inline var/spack/repos/builtin/packages/pegtl/package.py]: Done.

**Comment by junghans** [comment]
@adamjstewart: fixed

**Comment by junghans** [comment]
@adamjstewart: fixed

**Pull request merged**